main.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	stdlog "log"
+	"net/http"
+	_ "net/http/pprof"
+	"os"
+	"os/signal"
+	"reflect"
+	"syscall"
+	"time"
+
+	"github.com/go-kit/kit/server"
+	jsoncodec "github.com/go-kit/kit/transport/codec/json"
+
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+	"github.com/streadway/handy/cors"
+	"github.com/streadway/handy/encoding"
+	"golang.org/x/net/context"
+
+	kitlog "github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/metrics"
+	"github.com/go-kit/kit/metrics/expvar"
+	"github.com/go-kit/kit/metrics/prometheus"
+	"github.com/go-kit/kit/metrics/statsd"
+	"github.com/go-kit/kit/tracing/zipkin"
+	httptransport "github.com/go-kit/kit/transport/http"
+)
+
+func main() {
+	var (
+		debugAddr = flag.String("debug.addr", ":8000", "Address for HTTP debug/instrumentation server")
+		httpAddr  = flag.String("http.addr", ":8001", "Address for HTTP (JSON) server")
+	)
+	flag.Parse()
+
+	// `package log` domain
+	var logger kitlog.Logger
+	logger = kitlog.NewPrefixLogger(os.Stderr)
+	logger = kitlog.With(logger, "ts", kitlog.DefaultTimestampUTC)
+	kitlog.DefaultLogger = logger                     // for other gokit components
+	stdlog.SetOutput(kitlog.NewStdlibAdapter(logger)) // redirect stdlib logging to us
+	stdlog.SetFlags(0)                                // flags are handled in our logger
+	// `package metrics` domain
+	requests := metrics.NewMultiCounter(
+		expvar.NewCounter("requests"),
+		statsd.NewCounter(ioutil.Discard, "requests_total", time.Second),
+		prometheus.NewCounter(stdprometheus.CounterOpts{
+			Namespace: "congo",
+			Name:      "requests_total",
+			Help:      "Total number of received requests.",
+		}, []string{}),
+	)
+	duration := metrics.NewMultiHistogram(
+		expvar.NewHistogram("duration_nanoseconds_total", 0, 100000000, 3),
+		statsd.NewHistogram(ioutil.Discard, "duration_nanoseconds_total", time.Second),
+		prometheus.NewSummary(stdprometheus.SummaryOpts{
+			Namespace: "congo",
+			Name:      "duration_nanoseconds_total",
+			Help:      "Total nanoseconds spend serving requests.",
+		}, []string{}),
+	)
+
+	// `package tracing` domain
+	zipkinHost := "my-host"
+	zipkinCollector := loggingCollector{}
+	zipkinAddName := "CONGO" // is that right?
+	zipkinAddSpanFunc := zipkin.NewSpanFunc(zipkinHost, zipkinAddName)
```

Comment:
I would just leave the Zipkin stuff out for now. It's only useful if you already have a Zipkin infrastructure in place.